lib/railjet/bus.rb

```diff
@@ -52,6 +52,14 @@ def listen_to(event, &block)
         define_listeners(event, &block)
       end
 
+      def sidekiq_options(options = {})
+        if options.present?
+          @sidekiq_options = options
+        else
+          @sidekiq_options
```

Comment:
If you don't call `sidekiq_options` then `@sidekiq_options` returns `nil`, not sure if Sidekiq handles that, perhaps fallback to empty `{}`
```suggestion
          @sidekiq_options || {}
```

Reply:
it should not be a problem after 135add7

Reply:
Cool 👍